build/avalanchego-apis/admin-api.md

```diff
@@ -26,9 +26,9 @@ Assign an API endpoint an alias, a different endpoint for the API. The original
 admin.alias({endpoint:string, alias:string}) -> {success:bool}
 ```
 
-* `endpoint` is the original endpoint of the API. `endpoint` should only include the part of the endpoint after `/ext/`.
-* The API being aliased can now be called at `ext/alias`.
-* `alias` can be at most 512 characters.
+- `endpoint` is the original endpoint of the API. `endpoint` should only include the part of the endpoint after `/ext/`.
```

Comment:
nit: can you leave all of the `*` instead of changing them to `-` ?

Gitbook (the thing we use to host the docs) will just auto-revert them to `*`